src/code/bank20.asm

```diff
@@ -1857,70 +1857,95 @@ func_020_54F5::
     call CopyLinkTilesPair                        ; $555C: $CD $0A $1D
     ret                                           ; $555F: $C9
 
-Data_020_5560::
-    db   $08, $08, $0A, $0A
-
-Data_020_5564::
-    db   $01, $FF, $F0, $10
-
-Data_020_5568::
-    db   $00, $00, $03, $00
-
-Data_020_556C::
-    db   $01, $1F, $E0, $20
+; amount of BG tiles to be updated for next row / coloumn
+RegionUpdateTileAmount::
+.right:  db TILES_PER_COLOUMN - 1
+.left:   db TILES_PER_COLOUMN - 1
+.up:     db TILES_PER_ROW
+.down:   db TILES_PER_ROW
+
+; I am not absolute sure about that, but it is used as an offset
+TransitionOffsetTable::
+.right:  db $01
+.left:   db $FF
+.up:     db $F0
+.down:   db $10
+
+RegionOriginHighOffset::
+.right:  db $00
+.left:   db $00
+.up:     db $03
+.down:   db $00
+
+RegionOriginLowOffset::
+.right:  db $01
+.left:   db $1F
+.up:     db $E0
+.down:   db $20
```

Comment:
.label: or .label here?

Reply:
I'd say `.label:` (it seems the form without the semicolon is going to be deprecated in rgbds soon).

Reply:
ok then i will change it at the other place

Reply:
Local `.label` is not going to be deprecated, only `label`. But I would always add semicolons to labels just to be clear.

Reply:
`.label` is going to be depricated? So no local labels anymore?
What do you mean with semicolons?
.label ; comment
or
; comment
.label
or did you meant colon (:) ?

Reply:
done